internal/log/log.go

```diff
@@ -1,20 +1,20 @@
-package tfa
+package log
 
 import (
 	"os"
 
 	"github.com/sirupsen/logrus"
 )
 
-var log logrus.FieldLogger
+var Log logrus.FieldLogger
 
-func NewDefaultLogger() logrus.FieldLogger {
+func NewDefaultLogger(level, format string) logrus.FieldLogger {
```

Comment:
This function with exported `Log` variable is a non-standard setup. I'd split it to `NewDefaultLogger` that returns some default logger configuration and `NewLogger` for packages that need to configure own logger setup.

Reply:
@mhrabovcin yes, that was a hack :man_facepalming: , now I've got log.Log.blah() everywhere. I'll fix this today.